ecs-agent/tmds/server.go

```diff
@@ -0,0 +1,155 @@
+// copyright amazon.com inc. or its affiliates. all rights reserved.
```

Comment:
nit: capitalizing starting letters in the copyright blob

Reply:
err I will fix that.

Reply:
Updated